pytest_sphinx.py

```diff
@@ -70,10 +70,14 @@ def _is_doctest(config, path, parent):
 _OPTION_SKIPIF_RE = re.compile(r':skipif:\s*([^\n\'"]*)$')
 
 _DIRECTIVE_RE = re.compile(
-    r"""\s*\.\.\s
-    (testcode|testoutput|testsetup|testcleanup|doctest)
-    ::\s*([^\n'"]*)$
-    """, re.VERBOSE
+    r"""
+    \s*\.\.\s
+    (?P<directive>(testcode|testoutput|testsetup|testcleanup|doctest))
+    ::\s*
+    (?P<argument>([^\n'"]*))
+    $
+    """,
+    re.VERBOSE | re.MULTILINE,
```

Comment:
I think it in this change re.MULTILINE is not yet needed. Apart from that the change looks good! Nic!

Reply:
Good catch, removed!